services/getProductById.test.ts

```diff
@@ -0,0 +1,27 @@
+import { handler } from './getProductById';
+import { Context } from 'aws-lambda';
+
+const contextMock = {} as Context;
+
+describe('getProductsId microservice', () => {
```

Comment:
Microservice is equal to product service.

getProductsId is just handler inside of this microservice.

'getProductsId handler'